tools/RAiDER/delayFcns.py

```diff
@@ -124,9 +140,53 @@ def get_delays(stepSize, pnts_file, wm_file, interpType = '3D',
 #        pass
 
 #    return wet, hydro
+    time_elapse = (time.time() - t0)
+    time_elapse_hr = int(np.floor(time_elapse/3600.0))
+    time_elapse_min = int(np.floor((time_elapse - time_elapse_hr*3600.0)/60.0))
+    time_elapse_sec = (time_elapse - time_elapse_hr*3600.0 - time_elapse_min*60.0)
+    print("Delay estimation cost {0} hour(s) {1} minute(s) {2} second(s) using {3} cpu threads".format(time_elapse_hr,time_elapse_min,time_elapse_sec,cpu_num))
+#    sio.savemat('test.mat',{'wet_delay':wet_delay,'hydro_delay':hydro_delay})
+    sio.savemat('get_delays_time_elapse.mat',{'time_elapse':time_elapse})
     return wet_delay, hydro_delay
 
 
+
+def unpacking_hdf5_read(tup):
+    """
+        Like numpy.apply_along_axis(), but and with arguments in a tuple
+        instead.
+        This function is useful with multiprocessing.Pool().map(): (1)
+        map() only handles functions that take a single argument, and (2)
+        this function can generally be imported from a module, as required
+        by map().
+        """
+    
+    from pyproj import Transformer, CRS
+    
+    k, chunkInds, SP, SLV, in_shape, stepSize, ifWet, ifHydro, max_len, wm_file = tup
+    
+    # Transformer from ECEF to weather model
```

Comment:
It might be more efficient to create this outside the for-loop and pass a copy of the transformer with each tup.

Reply:
Based on my conversation with Yang, it looks like the transformer object must be created inside each process; otherwise it crashes.